app/src/main/java/com/ncnf/views/fragments/organization/OrganizationEventsFragment.java

```diff
@@ -0,0 +1,146 @@
+package com.ncnf.views.fragments.organization;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.Window;
+import android.widget.FrameLayout;
+
+import androidx.activity.OnBackPressedCallback;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.constraintlayout.widget.ConstraintLayout;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentTransaction;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.ncnf.R;
+import com.ncnf.adapters.EventListAdapter;
+import com.ncnf.models.Event;
+import com.ncnf.models.Organization;
+import com.ncnf.repositories.OrganizationRepository;
+import com.ncnf.views.activities.main.MainActivity;
+import com.ncnf.views.fragments.event.EventFragment;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import dagger.hilt.android.AndroidEntryPoint;
+
+import static com.ncnf.utilities.StringCodes.FRAGMENT_ORGANIZATION_TAG;
+
+@AndroidEntryPoint
+public class OrganizationEventsFragment extends Fragment {
+
+    @Inject
+    public OrganizationRepository organizationRepository;
+
+    private Organization organization;
+    private String uuid;
+
+    private RecyclerView recyclerView;
+    private EventListAdapter adapter;
+    private final List<Event> eventsList = new LinkedList<>();
+
+    public OrganizationEventsFragment() {}
+
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        setHasOptionsMenu(true);
+        //if it is null it should fail
+        this.uuid = this.getArguments().getString("organization_id");
```

Comment:
Isn't "organization_id" already defined in utilities ?

Reply:
You're right. This should be in a constant.